### PR TITLE
Update 2015-04-21-bubble-chart.md

### PR DESCRIPTION
**Pull request by MatthewHiscock**
Clarified language (with Marianne) and changed image names. @mariahhamel  @chriddyp  @NicoleGrondin  @mkcor

### CODE REVIEW

File: _posts/2015-04-21-bubble-chart.md [hunk 1]
```diff
@@ -21,33 +21,33 @@ actioncall: Make a Bubble Chart
...
+This is what the “Style” tab of the TRACES popover should look like for “All Traces (Scatter)”.We’ve set the “Size” field to scale the bubbles’ diameter, not area. And we’ve evened out the pixel-to-value ratio (the higher the value in the box, the smaller the bubbles will be). We’ve also increased the weight of the white bubble outlines. |  ![Style tab adjusted](/static/images/bubble-chart/style-tab-adjusted.png)
+Now, open the AXES popover in the toolbar. This is what the “Range” tab for the X Axis looks like. From looking at this y vs x bubble chart, it seems there could be an exponential relationship between y and x (y as a function of x). If that's the case, then there is a linear relationship between y and log(x). Indeed, if you use a log scale for the x axis, the bubbles appear more linear. |  ![Axes icon](/static/images/bubble-chart/axes-icon.png)  ![How to make a bubble chart online](/static/images/bubble-chart/log-option.png)
```
Comment (mkcor) [inline _posts/2015-04-21-bubble-chart.md]: Shouldn't it be **AXES**?

**Comment by mkcor** [comment]
I'm still unsure of the style consistency, because the same sentence writes 'x' and 'y' (where I guess we would want $x$ and $y$). But it's good enough for now.

+1

**Comment by mariahhamel** [comment]
@mkcor Do you mean the difference between 'x' and 'X' (lowercase/uppercase) for referring to the x variable? Or do you mean we should change to LaTeX (the newer tutorials I wrote use LaTeX). If you mean LaTeX, I'm guessing @MatthewHiscock doesn't know about this.

**Comment by mkcor** [comment]
I meant both ;) but haven't put thought into what should be favoured and why / when / how... If I knew specifically what we should go for, I would show @MatthewHiscock the basic LaTeX that's required. My intuition would be to write $x$ and the $x$-axis, so LaTeX all the way...

**Comment by MatthewHiscock** [comment]
I can confirm @matthewhiscock doesn't know about LaTeX. Happy to learn tho.

**Pull request merged**